build-tasks/templates/rule.template.js

```diff
@@ -8,8 +8,6 @@ import {AstUtils} from './utils/AstUtils';
 // use Utils instead of Underscore functions // TODO: delete comment
```

Comment:
Looks like changes unintentionally brought in from a recent PR; please revert.

_(if you fix the other comment I can go ahead and do this for you - it shouldn't block the PR)_

Reply:
I'll fix these up.

Reply:
Sorry, I wasn't very clear - no changes to this file should be in this PR (to keep PRs small and single-purpose). If you'd like to send them in, it should be discussed in a separate issue.